src/__tests__/components/Articles/__snapshots__/ProgressBar.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
why do we need the snapshots?

Reply:
honestly, we don't need it  and it's already removed